ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -89,7 +90,7 @@ Item {
     Loader {
         id: membershipRequests
 
-        property int nbRequests: root.communityData.pendingRequestsToJoin.count
+        property int nbRequests: root.communityData.pendingRequestsToJoin.count || 0
```

Comment:
why is this needed ?

Reply:
The count was `undefined` sometimes and showing a warning

Reply:
I'm going to have to disagree with your suggestion. It's pretty standard practice in JS to do `falsey || other` and your suggestion is just way longer. Unless you have very strong feelings about this, I'll leave it as is.